test/cppsuite/test_harness/workload/database_model.cxx

```diff
@@ -143,4 +145,12 @@ database::set_workload_tracking(workload_tracking *tracking)
     testutil_assert(_tracking == nullptr);
     _tracking = tracking;
 }
+
+void
+database::set_create_config(bool use_compression)
+{
+    _collection_create_config = use_compression ?
+      std::string(DEFAULT_FRAMEWORK_SCHEMA) + std::string(SNAPPY_BLK) :
```

Comment:
I think it can just be `DEFAULT_FRAMEWORK_SCHEMA SNAPPY_BLK`. The literals concatenate fine.

Reply:
Tried that, threw an error.

Reply:
Oh it's because `DEFAULT_FREAMEWORK_SCHEMA` is `const char*` not a `#define`.